homeworks/01-bubble-sort/src/typings/index.ts

```diff
@@ -0,0 +1,5 @@
+export type Col = {
```

Comment:
Why don't you like full readable names? `Col` is not obvious what it is at all. Do not shorten words, it's really hard to understand the logic.